CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.2] - 2019-11-19
+
+### Changed
+- UPS Freight: Pass TimeInTransitIndicator as a String rather than a Boolean
+
+## [0.4.1] - 2019-11-15
+
+Bugfix release
```

Comment:
What bug was fixed?

Reply:
Missing  `types.rb` file. Amended to changelog.